editor.planx.uk/src/@planx/components/Section/Public.tsx

```diff
@@ -153,71 +153,112 @@ export function SectionsOverviewList({
     const tagType = tagTypes[section];
 
-    const onClick =
-      tagType == TagType.Alert || tagType == TagType.Active
-        ? () => nextQuestion()
-        : () => {}; // no-op
-
-    return (
-      <Tag tagType={tagType} onClick={onClick} sectionTitle={sectionTitle}>
-        {section}
-      </Tag>
-    );
+    return <Tag tagType={tagType}>{section}</Tag>;
   };
 
   return (
     <DescriptionList>
-      {Object.entries(sectionNodes).map(([sectionId, sectionNode]) => (
-        <SectionRow key={sectionId}>
-          <SectionTitle>
-            {showChange &&
-            sectionStatuses[sectionId] === SectionStatus.Completed ? (
-              <Link
-                onClick={() => changeFirstAnswerInSection(sectionId)}
-                component="button"
-              >
-                <Typography
-                  variant="subtitle1"
-                  component="h4"
-                  color="primary"
-                  align="left"
-                >
-                  <span style={visuallyHidden}>{`Change `}</span>
-                  <strong>{sectionNode.data.title}</strong>
-                </Typography>
-              </Link>
-            ) : (
-              <Typography variant="subtitle1" component="h4" color="inherit">
-                <strong>{sectionNode.data.title}</strong>
-              </Typography>
-            )}
-            <ReactMarkdownOrHtml
-              source={sectionNode.data.description}
-              openLinksOnNewTab
-            />
-          </SectionTitle>
-          <SectionState>
-            {getTag(sectionStatuses[sectionId], sectionNode.data.title)}
-          </SectionState>
-        </SectionRow>
-      ))}
+      {Object.entries(sectionNodes).map(([sectionId, sectionNode]) => {
+        const status = sectionStatuses[sectionId];
+        const clickHandler = getClickHandler(sectionId, status);
+        const isClickable = clickHandler !== undefined;
+        const statusId = `section-${sectionId}-status`;
+        const hintId = `section-${sectionId}-hint`;
+
+        return (
+          <SectionRowWrapper
+            key={sectionId}
+            isClickable={isClickable}
+            onClick={
+              isClickable
+                ? (e) => {
+                    if (
+                      e.target === e.currentTarget ||
+                      !e.currentTarget
+                        .querySelector("a")
+                        ?.contains(e.target as Node)
+                    ) {
+                      clickHandler?.();
+                    }
+                  }
+                : undefined
+            }
+          >
+            <SectionContent>
+              <SectionDescription>
+                {isClickable ? (
+                  <SectionTitleLink variant="subtitle1" component="h4">
+                    <Link
+                      href="#"
+                      onClick={(e) => {
+                        e.preventDefault();
+                        clickHandler?.();
```

Comment:
I _think_ this is redundant. As this is within a clickable element, and we're not calling `e.stopPropogation()`, the event will bubble up and just trigger the click handler there.

Reply:
This probably simplifies the check above on the `SectionRowWrapper` as well.